### PR TITLE
Feat/cfe-272 contract delegation

### PR DESCRIPTION
**Pull request by jennieramida**
## Describe your changes
cfe-272

### CODE REVIEW

File: src/lib/components/delegations/DelegationInfo.tsx [hunk 1]
```diff
@@ -43,16 +47,8 @@ export const DelegationInfo = ({
...
-            variant="h6"
-            as="h6"
-            display="flex"
-            alignItems="center"
-            height={{ base: "29px", md: "auto" }}
-          >
-            Delegations
-          </Heading>
+        <Flex direction="column" gap={4} mt={{ base: 4, md: 0 }}>
+          {tableTitle}
```
Comment (songwongtp) [inline src/lib/components/delegations/DelegationInfo.tsx:51]: can we just use `<TableTitle title="Delegations" mb={0} showCount={false} />` here?

File: src/lib/pages/contract-details/index.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { observer } from "mobx-react-lite";
 import { useRouter } from "next/router";
 import { useCallback, useEffect } from "react";
 
+import { DelegationsSection } from "../../components/delegations";
+import { AssetsSection } from "../account-details/components/asset";
```
Comment (songwongtp) [inline src/lib/pages/contract-details/index.tsx:15]: 1. move `asset` folder from `account-details/components` to `lib/components`
2. rename the folder to `asset-section`

File: src/lib/pages/contract-details/index.tsx [hunk 3]
```diff
@@ -59,6 +61,10 @@ const ContractDetailsBody = observer(
       isLoading,
     } = useContractData(contractAddress);
 
+    // const { totalData } = useBalanceInfos(contractAddress);
```
Comment (songwongtp) [inline src/lib/pages/contract-details/index.tsx:64]: remove

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/contract-details/index.tsx (hunk 2 above)
Comment (jennieramida) [reply src/lib/pages/contract-details/index.tsx:15]: already move in another pr

**Review by songwongtp — CHANGES_REQUESTED** [review]

**Review by songwongtp — APPROVED** [review]

**Pull request merged**